assets/scss/modal.scss

```diff
@@ -27,3 +42,7 @@
     padding: 1rem 0;
```

Comment:
This padding isn't needed anymore now. It's causing extra space in the user form modal `/?path=/story/user-collection--modal`

Reply:
maybe the footer only needs padding on the top.

Reply:
good spot I've removed padding except for top